ApollosProject/apollos-apps
# Fix: focus states on TV auth screens

## ✏️ Description

This PR fixes a couple of issues with focused states on the authentication on the Tv app

## 🔬 To Test
1. pull down PR
2.run TV on sim 
3. check focus state colors on the OTP input and the phone/email input. 

## 📸 Screenshots
Forgot to grab before I closed the sim. Can spin back up if needed.
